package/MDAnalysis/analysis/base.py

```diff
@@ -134,7 +134,7 @@ def _setup_frames(self, trajectory, start=None, stop=None, step=None):
         if interval == 0:
```

Comment:
Do also add a versionchanged to `run` to document this new behaviour (i.e. temporary overriding of `__init__` set `verbose`).